controllers/rdsinventory_controller.go

```diff
@@ -460,6 +463,9 @@ func (r *RDSInventoryReconciler) Reconcile(ctx context.Context, req ctrl.Request
 		waitForAdoptedResource := false
 		for i := range adoptedDBInstanceList.Items {
```

Comment:
This loop can be simplified to:
for _, adoptedDBInstance := range adoptedDBInstanceList.Items { ... }

Reply:
OK